product/clean_samples/add_fulfillment.js

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```

Comment:
you have already created add_remove_fulfillment.js,  I suppose here should be set_inventory methods?

Reply:
These are separate samples for add and remove fulfillment